openfisca_aotearoa/variables/acts/social_security/disability_allowance/disability_allowance.py

```diff
@@ -0,0 +1,220 @@
+        no_partners = numpy.logical_not(persons("social_security__in_a_relationship", period))
+        income_within_limit = persons("social_security__income", period) <= parameters(period).disability_allowance.income_limits.clauses["clause_10"]
+        without_dependant_child = persons("social_security__dependent_children", period) == 0
+        return person_aged_16_or_17 * no_partners * income_within_limit * without_dependant_child
+
+
+class disability_allowance__income_limit_clause_11(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Does the person earn below the Disability Allowance Income Limit as per clause 11?"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784890.html"
+
+    def formula_2018_11_26(persons, period, parameters):
+        person_not_aged_16_or_17 = (persons("age", period.start) < 16) + (persons("age", period.start) > 17)
+        no_partners = numpy.logical_not(persons("social_security__in_a_relationship", period))
+        without_dependant_child = persons("social_security__dependent_children", period) == 0
+        not_a_child = numpy.logical_not(persons.has_role(entities.Family.CHILD))
+        income_within_limit = persons("social_security__income", period) <= parameters(period).disability_allowance.income_limits.clauses["clause_11"]
+        return person_not_aged_16_or_17 * no_partners * without_dependant_child * income_within_limit * not_a_child
+
+
+class disability_allowance__income_limit_clause_12(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Does the person earn below the Disability Allowance Income Limit as per clause 12?"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784890.html"
+
+    def formula_2018_11_26(persons, period, parameters):
+        in_relationship = persons("social_security__in_a_relationship", period)
+        income_within_limit = persons("disability_allowance__family_income", period) <= parameters(period).disability_allowance.income_limits.clauses["clause_12"]
+        return in_relationship * income_within_limit
+
+
+class disability_allowance__income_limit_clause_13(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Does the person earn below the Disability Allowance Income Limit as per clause 13?"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784890.html"
+
+    def formula_2018_11_26(persons, period, parameters):
+        sole_parent = persons("social_security__sole_parent", period)
+        only_one_child = (persons("social_security__dependent_children", period) == 1)
```

Comment:
Same as before, by testing _1_ over a boolean variable, you're actually checking for truthyness.

```suggestion
principal = persons.has_role(entities.Family.PRINCIPAL)
dependent_children = persons("social_security__dependent_children", period)
only_one_child =  principal * (persons.family.sum(dependent_children, role = entities.Family.CHILD) == 1)
```